src/OpenSSL/SSL.py

```diff
@@ -699,8 +714,98 @@ def set_default_verify_paths(self):
 
         :return: None
         """
+        # This function will attempt to load certs from both a cafile and
+        # capath that are set at compile time. However, it will first check
+        # environment variables and, if present, load those paths instead
```

Comment:
This comment feels like it's in the wrong place. It checks the default paths _before_ manualyl doing the env vars.